modules/nf-core/star/genomegenerate/tests/main.nf.test

```diff
@@ -0,0 +1,40 @@
+nextflow_process {
+
+    name "Test Process STAR_GENOMEGENERATE"
+    script "../main.nf"
+    process "STAR_GENOMEGENERATE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "star"
+    tag "star/genomegenerate"
+
+    test("homo_sapiens") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test_fasta' ],
+                    [file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)]
+                ])
+                input[1] = Channel.of([
+                    [ id:'test_gtf' ],
+                    [file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)]
+                ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert path(process.out.index.get(0).get(1)).exists() },
```

Comment:
I personally like to snapshot the name of files that I want to check the existence of. That way you also check that the output naming stays correct. (Just a suggestion, this is definitely fine too)

Reply:
`file(process.out.index[0][1]).name` would do that BTW